emails/tests/views_tests.py

```diff
@@ -1,83 +1,126 @@
 from datetime import datetime, timezone
 import json
 import os
-from unittest import skip
 from unittest.mock import patch
 
-from django.conf import settings
 from django.contrib.auth.models import User
-from django.test import (
-    Client,
-    override_settings,
-    TestCase,
-)
+from django.test import override_settings, TestCase
 
+from allauth.socialaccount.models import SocialAccount
 from model_bakery import baker
 
 from emails.models import (
     address_hash,
     DeletedAddress,
+    Profile,
     RelayAddress,
 )
-from emails.views import _get_address
+from emails.views import _get_address, _sns_notification
 
-TEST_DOMAINS = {'RELAY_FIREFOX_DOMAIN': 'default.com', 'MOZMAIL_DOMAIN': 'test.com'}
```

Comment:
Removed this in favor of the new `get_domains_from_settings` function that returns these test values while in test mode.